Sources/AccountSDKIOSWeb/Client/Client.swift

```diff
@@ -224,21 +225,66 @@ public class Client: CustomStringConvertible {
     func destroySession() {
         sessionStorage.remove(forClientId: configuration.clientId)
     }
+    
+    func switchToSharedKeychain(appIdentifierPrefix: String?, completion: @escaping (Result<Void, Error>) -> Void) {
+        guard let appIdentifierPrefix = appIdentifierPrefix, !appIdentifierPrefix.isEmpty else {
+            completion(.failure(SimplifiedLoginManager.SimplifiedLoginError.appIdentifierPrefixNotProvided))
+            return
+        }
```

Comment:
Aha now I see. We only switch to shared keychain if there is appIdentifier prefix.

Reply:
and also if current access group is not shared keychain access group